src/analysis/properties.rs

```diff
@@ -174,12 +174,33 @@ pub fn get_type_default_value(env: &Env, type_tid: library::TypeId, type_: &libr
                 Fundamental::Utf8 => some("None::<&str>"),
                 Fundamental::Float => some("&0f32"),
                 Fundamental::Double => some("&0f64"),
+                Fundamental::None => some("&0"),
```

Comment:
I just copy pasted all the fundamentals, if you think there are some we should skip I'm fine with removing them.